setup/docker/slurm/start_slurm.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Set the container's hostname in slurm.conf
+sed "s^{{ hostname }}^`hostname`^g" /etc/slurm/slurm.conf.in >  /etc/slurm/slurm.conf
```

Comment:
You could also copy the file directly to `/etc/slurm/slurm.conf` and `sed -i "[pattern]" /etc/slurm/slurm.conf`

Reply:
I debated that, but after needing to debug something once, I decided I wanted to have the original file available in case we need to continue debugging.

Reply:
Okay, makes sense.